src/components/playback/playbackmanager.js

```diff
@@ -2277,6 +2278,8 @@ class PlaybackManager {
                 playOptions.isFirstItem = false;
             } else {
                 playOptions.isFirstItem = true;
+                // resetting the aspectRatio from a previous theme video playback
+                if (!playOptions.aspectRatio) playOptions.aspectRatio = 'auto';
             }
```

Comment:
This code (`isFirstItem`) is broken (never worked?), imo.

Tested on a playlist. When you click next, `playOptions.isFirstItem` is always `undefined` because `playOptions` goes from `getDefaultPlayOptions`:
https://github.com/jellyfin/jellyfin-web/blob/c9d47f0a0b40f96116db9cfb8274881a0fae0e5c/src/components/playback/playbackmanager.js#L3027
When you click previous, `isFirstItem` is set (`true`) and changes from `true` to `false`. It looks like `playOptions` is only stored in the first item. :confused: 
When you click next again, it is `undefined` again.
When you click previous again, `isFirstItem` is set (`false`) and changes from `false` to `true`.
:spaghetti: :man_facepalming:

We need to fix `isFirstItem` first.

Reply:
Heh, I tested skipping to the next item, but didn't think of going back to previous :D